File: java/processor.java
class Processor{
    //atribut
    private String name;
    private int price;
     
    //konstruktor 
    Processor(){}

    //setter & getter name
    public void setname(String name){
        this.name = name;
    }
    public String getname(){
        return name;
    }

    //setter & getter price
    public void setprice(int price){
        this.price = price;
    }
    public int getprice(){
        return price;
    }

    void printProcessor(){
        System.out.println("Processor   : " + this.name);
    }
}

File: c++/processor.cpp
class Processor{
    private:
        //atribut
        string name;
        int price;

    public:
        //konstruktor
        Processor(){}
        Processor(string n, int p){
            this->name = n;
            this->price = p;
        }

        //setter & getter
        void setName(string n){
            this->name = n;
        }
        string getName(){
            return this->name;
        }

        void setPrice(int p){
            this->price = p;
        }
        int getPrice(){
            return this->price;
        }

        void printProcessor(){
            cout << "Processor   : " << this->name << endl;
        }

        //destruktor
        ~Processor(){
        }
};
